src/java.base/share/classes/java/nio/file/FileTreeIterator.java

```diff
@@ -36,15 +36,15 @@
 /**
  * An {@code Iterator} to iterate over the nodes of a file tree.
  *
- * <pre>{@code
+ * {@snippet lang=java :
```

Comment:
This is a JDK internal class so the example won't show up in the API docs, just FYI.

Reply:
> Also some changes were made to package private classes which do not appear in the nominal javadoc.

Right, that was the motivation for my first comment after opening the PR which is excerpted here.